android-components/components/ui/icons/src/main/res/values/colors.xml

```diff
@@ -5,4 +5,7 @@
 
 <resources>
     <color name="mozac_ui_icons_fill">#FFFFFF</color>
+
+    <color name="mozac_ic_star_filled">#000000</color>
```

Comment:
Maybe add a comment to specify that these are used in mozac_ic_star_one_half_fill_20